src/core/engines/Cesium/Feature/Tileset/hooks.ts

```diff
@@ -95,13 +95,18 @@ type CachedFeature = {
 
 const MAX_NUMBER_OF_CONCURRENT_COMPUTING_FEATURES = 5000;
 
-type StyleProperty<N = string> = { name: N; convert?: "color" | "vec2" | "vec4" };
+type StyleProperty<N = string> = {
+  name: N;
+  convert?: "color" | "colorFunctionString" | "booleanString" | "vec2" | "vec4";
+};
 
 const COMMON_STYLE_PROPERTIES: StyleProperty<"color" | "show">[] = [
   { name: "color", convert: "color" },
   { name: "show" },
 ];
-const MODEL_STYLE_PROPERTIES: StyleProperty<"pointSize" | "meta">[] = [
+const MODEL_STYLE_PROPERTIES: StyleProperty<"color" | "show" | "pointSize" | "meta">[] = [
+  { name: "color", convert: "colorFunctionString" },
+  { name: "show", convert: "booleanString" },
```

Comment:
```suggestion
  { name: "color", convert: "color" },
  { name: "show" },
```

I think it should work, but is there problem?

Reply:
Since this object is used as param of `new Cesium3DTileStyle` I think the `Color` type is not supported? If use Color cesium will crash with `TypeError: style.color.evaluateColor is not a function`